modules/oaiharvest/lib/oai_harvest_daemon.py

```diff
@@ -572,36 +572,47 @@ def remove_duplicates(harvested_file_list):
     """
     Go through a list of harvested files and remove any duplicate records.
     """
+    from copy import deepcopy
+    import lxml.etree as etree
```

Comment:
> WRT 1.2 I had a separate pull request for master as the code changed there a bit.

Yes, I noticed.  If we merge this functionality into maint-1.1, then the other PR will come later.  If we don't merge this functionality into maint-1.1, then I can simply merge the other PR, and we are done.  (Because there we don't have to worry about lxml optional-or-mandatory, but we can make it mandatory.)  Hence my wondering whether you would like to include this into maint-1.1?  If yes, please try to make try/expect gentler.  If not, then I'll merge into 1.2 only, and we are done.